packages/larva-patterns/modules/vlanding-video-card/vlanding-video-card.twig

```diff
@@ -6,9 +6,9 @@
 		data-video-showcase-dek="{{ vlanding_video_card_link_showcase_dek_data_attr }}"
 		data-video-showcase-title="{{ vlanding_video_card_link_showcase_title_data_attr }}"
 		data-video-showcase-permalink="{{ vlanding_video_card_link_showcase_permalink_data_url }}"
-		{% if vlanding_video_card_link_showcase_type_data_attr == 'youtube' %}
```

Comment:
Can the controller be responsible for providing the `aria-controls` value as well as the `vlanding_video_card_link_showcase_type_data_attr`?

So this would be updated to:
```
{% if vlanding_video_card_link_aria_controls %}
			aria-controls="{{ vlanding_video_card_link_aria_controls }}"
{% endif %}
```

Reply:
And in the README for this module, make a note of the values that should be provided in each chase - e.g. 'youtube' along with aria-controls for 'youtubeContainer'